Support for :source [file] in cypher-shell

pontusmelke opened the pull request:
Adds support for the command `:source` where you can in interactive mode
run statements from an external file.

NOTE: builds on top of https://github.com/neo4j/cypher-shell/pull/167 just last commit to review here

changelog: Support interactively executing Cypher statements from a file via `:source statements.cypher`.

fickludd commented on cypher-shell/src/integration-test/java/org/neo4j/shell/MainIntegrationTest.java:
```diff
@@ -203,6 +208,75 @@ public void shouldHandleInvalidCypherFromFile() throws Exception {
+    }
+
+    @Test
+    public void shouldReadHandleInvalidCypherStatementsFromFileInteractively() throws Exception {
```
should~Read~HandleInvalidC...

pontusmelke committed:
Support for :source [file] in cypher-shell

Adds support for the command `:source` where you can in interactive mode
run statements from an external file.

fickludd reviewed (approved):
This is great stuff. Like the testing and impl.

pontusmelke committed:
Code Review: typo
